README.md

```diff
@@ -79,6 +79,17 @@ you want to learn about a specific release, check out [the release list].
 
 [the release list]: https://github.com/iced-rs/iced/releases
 
+The majority of documentation currently follows the master branch,
+it may be suggested to use git referenced dependency in your Cargo.toml file.
+
+```toml
+[dependencies.iced]
+git = "https://github.com/iced-rs/iced.git"
+branch = "master"
+features = ["canvas", "tokio", "debug", "async-std", "lazy", "webgl"]
```

Comment:
Should these be the set of dependencies we recommend? Might be better to not include them at all.

Reply:
Thanks for the edit!
I do believe it would be helpful for anyone else who may be approaching the lib completely green to it, for them to be aware of this nuance, at least until 0.13 is released and all the docs align with a release. It could potentially help with adoption in the meantime.